be/src/storage/task/engine_checksum_task.cpp

```diff
@@ -71,12 +71,6 @@ Status EngineChecksumTask::_compute_checksum() {
         return Status::NotFound(fmt::format("Not found tablet: {}", _tablet_id));
     }
 
-    if (tablet->updates() != nullptr) {
```

Comment:
It will scan all of the data to calculate the checksum. 
1. It maybe costing for PK table.
2. In the night, it can't finish calculate for all ingested tables.